### DISCUSSION TITLE
Pivot Table  //  orientation & collapsing

### DISCUSSION THREAD

**Discussion by m0styr** [dfinke/ImportExcel#1524] [Q&A]
Hi **ImportExcel** community :-)

please, I would like to ask you for help about formatting a pivot table.
I need change change orientation a also collapse all sub-rows.. see attached picture.
Is possible to do it via Export-Excel and Add-PivotTable function, or somehow else?

Thanks in advance for any help.

m0styr

![image](https://github.com/dfinke/ImportExcel/assets/48366970/93594def-33be-4b9d-af75-e76c7756d9c8)

**Comment by dfinke** [comment]
Thanks for using ImportExcel.

There are a few pivot table functions. On the `New-PivotTableDefinition`  there are:
- PivotRows
- PivotData
- PivotColumns

Which let you shape your data. There is also `Add-PivotTable`, take a look at the params on that.

Finally, check the examples: https://github.com/dfinke/ImportExcel/tree/master/Examples/PivotTable
